frontend/summary/summary/ExploreHeatmap.js

```diff
@@ -178,7 +177,7 @@ class ExploreHeatmap extends BaseVisual {
             caption = new SmartTagContainer(captionDiv),
             $plotDiv = $("<div>"),
             callback = resp => {
-                if (resp.dataset) {
+                if (resp.dataset || resp.error) {
```

Comment:
i changed how this was implemented; and just  carried w/ the standard path even if an error was logged, just because it looks like they were generally the same so we didnt have as much duplicate code